### PR TITLE
Add JMS Source

### LINKED ISSUE #70: Port JMS as s-c-s source
As a developer, I'd like to move *jms* module from XD to s-c-s-m repo, so I can use it as source to build streaming pipeline. 

**Acceptance:**
* *jms* as source module is available in *spring-cloud-stream-modules* repo 
* maven repo includes *jms* module with respective group/artifact/version detail

### PR DESCRIPTION
**Pull request by garyrussell**
Resolves #70

### CODE REVIEW

File: jms-source/src/main/java/org/springframework/cloud/stream/module/jms/source/JmsSourceApplication.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.jms.source;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * A main application that can be used to run the TCP Source as a standalone app.
```
Comment (artembilan) [inline jms-source/src/main/java/org/springframework/cloud/stream/module/jms/source/JmsSourceApplication.java:23]: Copy/paste artefact :smile: 
s/TCP/JMS

Comment (garyrussell) [inline jms-source/src/main/java/org/springframework/cloud/stream/module/jms/source/JmsSourceApplication.java:23]: rats! I thought I'd found all those.

**Comment by artembilan** [comment]
That's all.
I'm fine to merge it and fix that simple issue in the JavaDocs.
Unless there is something else to discuss about "provider" concern...

CC @mbogoevici

**Comment by artembilan** [comment]
Merged as https://github.com/spring-cloud/spring-cloud-stream-modules/commit/6eb70ab82bd7b576444e8a061ebb872be2912e6d

**Pull request closed without merging**